web/lib/supabase/group.ts

```diff
@@ -123,3 +124,21 @@ export async function searchUserInGroup(
     'member_id'
   ).slice(0, limit)
 }
+
+export async function getGroupPrivacyBySlug(groupSlug: string) {
```

Comment:
General comment: We should probably switch to using group ids everywhere instead of having two ways to identify groups.

Oh, I see this is used from the group page when reading the url slug. That's the one acceptable usage haha.